docs/integration/squirrel.txt

```diff
@@ -64,4 +63,5 @@ and scripts.
 
 .. _SQuirreL: http://www.squirrelsql.org
 .. _Crate JDBC: https://crate.io/docs/projects/crate-jdbc/stable/
-.. _Crate JDBC standalone jar: https://bintray.com/crate/crate/crate-jdbc/view/files/io/crate/crate-jdbc-standalone
\ No newline at end of file
+.. _Crate JDBC standalone jar:
```

Comment:
does this work correctly with sphinx if there are line breaks here? (is the doc still rendered correctly?)